{"type": "request", "data": {"timestamp": 1740495970.735332, "service": "AIService", "operation": "extract_wahrheit_data", "success": false, "duration": 4.208284139633179, "metadata": {"error_type": "unknown"}}, "system_uptime": 5.070760250091553}
{"type": "retry", "data": {"timestamp": 1740495970.7354822, "service": "AIService", "operation": "extract_wahrheit_data", "attempt": 1, "error_message": "'str' object has no attribute 'model_dump'", "metadata": {"error_type": "unknown"}}, "system_uptime": 5.0709099769592285}
{"type": "request", "data": {"timestamp": 1740495975.095766, "service": "AIService", "operation": "extract_wahrheit_data", "success": false, "duration": 2.353052854537964, "metadata": {"error_type": "unknown"}}, "system_uptime": 9.431194067001343}
{"type": "retry", "data": {"timestamp": 1740495975.095932, "service": "AIService", "operation": "extract_wahrheit_data", "attempt": 2, "error_message": "'str' object has no attribute 'model_dump'", "metadata": {"error_type": "unknown"}}, "system_uptime": 9.431360006332397}
{"type": "error", "data": {"timestamp": 1740495975.096193, "service": "FileProcessor", "operation": "extract_wahrheit_data", "error_message": "Failed to process extract_wahrheit_data after 2 attempts. Last error: 'str' object has no attribute 'model_dump'", "metadata": {"total_duration": 8.569738864898682}}, "system_uptime": 9.431621074676514}
{"type": "error", "data": {"timestamp": 1740495975.09653, "service": "PackingListGenerator", "operation": "generate_packing_list", "error_message": "Error processing wahrheit data: Failed to process extract_wahrheit_data after 2 attempts. Last error: 'str' object has no attribute 'model_dump'", "metadata": {"total_duration": 8.570305824279785}}, "system_uptime": 9.431957006454468}
